SAMtags.tex

```diff
@@ -317,11 +320,29 @@ \subsection{Barcodes}
 \subsection{Original data}
 
 \begin{description}
+\item[OA:Z:\tagregex{{\tt (}\sf{RNAME}{\tt,\,}\sf{POS}{\tt,\,}\emph{strand}{\tt,\,}\sf{CIGAR}{\tt,\,}\sf{MAPQ}{\tt,\,}\sf{NM}{\tt; )}+}]
+The tag encodes the original alignment information of the record prior to realignment or unalignment by subsequent tool. 
+Here {\sf strand} encodes the 0x10 bit of {\sf FLAG}---{\sf SEQ} being reverse complemented---such that it is `+' if the bit is unset and `-' if the bit is set,
+Resulting string is the result of joining the 6 values with commas `,' (no additional spaces).
```

Comment:
"Resulting" here shouldn't start with a capital letter, or change the previous comma to a full-stop and use "The resulting string".